app/views/bathrooms/index.html.erb

```diff
@@ -1,13 +1,21 @@
+<%# you can use `content_for :head` to add this to head %>
+<%# as-is this is rendered in the <body> %>
+<%# https://guides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method %>
+<%# this is also already defined in <head> %>
 <head>
   <link href="https://api.mapbox.com/mapbox-gl-js/v3.2.0/mapbox-gl.css" rel="stylesheet">
   <script src="https://api.mapbox.com/mapbox-gl-js/v3.2.0/mapbox-gl.js"></script>
 </head>
+
+<%# we only have 1 body %>
 <body>
+  <%# could we add this to application.html.erb so it appears everywhere? %>
```

Comment:
I did put it in application.html.erb at first, but I didn't want it on my landing page specifically, so that's why I took it out. Is there a better way to do that?